packages/backend/package.json

```diff
@@ -5,7 +5,7 @@
   "types": "./src/index.ts",
   "main": "./dist/index.js",
   "scripts": {
-    "start": "nodemon -r dotenv/config ./src/index.ts dotenv_config_path=../../.env dotenv_config_debug=true",
+    "start": "ts-node -r dotenv/config ./src/index.ts dotenv_config_path=../../.env dotenv_config_debug=true",
```

Comment:
@eyalatox shouldn't you add ts-node in devDependencies , no we can be sure than it is available after `npm install` ?

Reply:
right, I'll add it

Reply:
ts-node isn't doing watch... why did you replace `nodemon`?